package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.0.1",
```

Comment:
Bumps the version to 1.0.0?

Reply:
Do we want to publish the packages under phala org?

Reply:
No, we only update this field internally and keep it versioned.